src/datascience-ui/data-explorer/reactSlickGrid.tsx

```diff
@@ -685,7 +667,8 @@ function readonlyCellEditor(this: any, args: any) {
         // to move their cursor within the focused input element
         if (
             (e.keyCode === KeyCodes.LeftArrow && cursorPosition > 0) ||
-            (e.keyCode === KeyCodes.RightArrow && cursorPosition < textLength - 1)
+            (e.keyCode === KeyCodes.RightArrow && cursorPosition < textLength - 1) ||
+            (e.ctrlKey && e.keyCode === KeyCodes.X)
```

Comment:
Fixing a problem that Rich found where ctrl+x deletes the contents of the focused cell. Not destructive (the old contents get restored as soon as cell focus is lost), just a very goofy UX